### ISSUE TITLE
Add "?orgId=N" everywhere

### ISSUE THREAD

**Issue by gelicia** [grafana/grafana#82723]
### Discussed in https://github.com/grafana/grafana/discussions/67024

<div type='discussions-op-text'>

<sup>Originally posted by **thomasbromkapsdata** April 21, 2023</sup>
Hello,
using Grafana with multiple organizations, i found very annoying that “?orgId=N” is not in every URL. By example, it appear well for dashboards but not for folders or datasources.
So it’s really easy to share a dashboard but not a folder or anything else: the link will only work well if the other user (or yourself going back to this link later) is already logged in the right organization or if you add manually the right orgId.
Thank you.</div>

**Comment by Invisi** [comment]
Hey there,

`orgId` for dashboard URLs has been missing since version 11.3.0. Up until 11.2.3 it was appended/inserted via [initDashboard.initDashboard](https://github.com/grafana/grafana/blob/933c83e373929bc767a55cbcf3996b2b07eaa681/public/app/features/dashboard/state/initDashboard.ts#L214), called from [DashboardPage.initDashboard](https://github.com/grafana/grafana/blob/933c83e373929bc767a55cbcf3996b2b07eaa681/public/app/features/dashboard/containers/DashboardPage.tsx#L160).

This regression was likely introduced during the migration to Grafana Scenes, causing `initDashboard.initDashboard` to not be called anymore unless disabled via feature flag/query param. Easily replicated by appending &scenes=false. ([ref](https://github.com/grafana/grafana/blob/d9455ff7db73b694db7d412e49a68bec767f2b5a/public/app/features/dashboard/containers/DashboardPageProxy.tsx#L22))

URL parameters like `from`, `to`, and `timezone` are still appended via the [`_urlSync` property](https://github.com/grafana/scenes/blob/8740f284bc40912553a0726b211dca1742d255fd/docusaurus/docs/url-sync.md?plain=1#L34), however there is no such `SceneObjectUrlSyncConfig` for `orgId`.


TL;DR: Since Grafana Scenes, there has been no orgId at all.


Related: https://github.com/grafana/scenes/issues/762, #91781
